docs/source/changelog.rst

```diff
@@ -1,6 +1,97 @@
 Changelog
 =========
 
+.. _v2024.8.2:
+
+2024.8.2
+--------
+
+Automatically choose P2P rechunking where applicable
+""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+TODO (placeholder)
+
+New shuffle API for Dask Arrays
+"""""""""""""""""""""""""""""""
+
+Dask added a shuffle-API to Dask Arrays. This API allows for shuffling the data
+along a single dimension. It will ensure that every group of elements along this
+dimension are in exactly one chunk. This is a very useful operation for GroupBy-Map
+patterns in Xarray. See :py:func:`~dask.array.Array.shuffle` for more information
+and API signature.
+
+See :pr:`11267`, :pr:`11311` and :pr:`11326` by `Patrick Hoefler`_ for more details.
+
+New blockwise_reshape API for Dask Arrays
+"""""""""""""""""""""""""""""""""""""""""
+
+Reshaping a Dask Array oftentimes creates a very complicated computations with rechunk
+operations in between because Dask respect the C ordering of the Array by default. This
+ensures that the resulting Dask Array is returned in the same order as the
+corresponding NumPy Array. However, this can lead to very inefficient computations.
+
+The new ``blockwise_reshape`` API allows for reshaping a Dask Array
+in embarassingly parallel fashion. This is useful when you don't care about the order of
+the resulting Array, i.e. if a reduction is applied to the array or if the reshaping
+is only temporary.
```

Comment:
Food 4 thought: I'ver personally come to the conclusion that it might be better to put the user benefit of changes into the foreground in these changelog entries (where applicable). For this entry, I think it might be more interesting for the user the order was switched around and read something like this:

> For cases where you just want to reshape but don't care about ordering (e.g., Xarray stuff), `blockwise_reshape` now comes to the rescue! It's embarrasingly-parallel and thus much less complex than `reshape` which has to jump through hoops to ensure correct ordering which results in a complex-and-difficult-to-compute graph.
> 
> Here's an example.
> 
> Here's more technical implementation stuff.
> 

Feel free to ignore, after all these are changelog notes, no clue if somebody even reads these.